main.css

```
@@ -47,6 +47,21 @@ body {
 	word-spacing: 0.1rem;
 }
 
+::-webkit-scrollbar {
+    width: 12px;
```

let's make it narrower
```suggestion
    width: 6px;
```